src/project/mod.rs

```diff
@@ -156,16 +156,28 @@ impl Project {
         &self.manifest.parsed.project.version
     }
 
+    /// Returns the description of the project
+    pub fn description(&self) -> &Option<String> {
+        &self.manifest.parsed.project.description
+    }
+
     /// Returns the root directory of the project
     pub fn root(&self) -> &Path {
         &self.root
     }
 
     /// Returns the pixi directory
-    pub fn environment_dir(&self) -> PathBuf {
+    pub fn pixi_dir(&self) -> PathBuf {
         self.root.join(consts::PIXI_DIR)
     }
 
+    /// Returns the environment directory
+    pub fn environment_dir(&self) -> PathBuf {
+        self.root
+            .join(consts::PIXI_DIR)
+            .join(consts::ENVIRONMENT_DIR)
+    }
```

Comment:
@ruben-arts pinging for visiblity.

Is that the correct way to set those getters?

Reply:
You could reuse the `pixi_dir()` if that is what you mean?

Reply:
I just wanted to check that was correct. But good idea to use `pixi_dir()` here.